docker-container/launch-magento.sh

```diff
@@ -0,0 +1,26 @@
+# Create Docker Envrioment
+set -x
+source config.sh
+rm -rf ../../magento-cloud/bolt-magento2
+cp -R $boltRepo  ../../magento-cloud/
+if [ "$localRepo" = true ] ; then
+    python compose-format.py  "*@dev" "./bolt-magento2" "path"
+else
+    python compose-format.py $boltBranch "./bolt-magento2" "vcs"
+fi
+cd ../../magento-cloud
+composer install
+if [ "$buildCompose" = true ] ; then
+    yes | ./vendor/bin/ece-tools docker:build
+    python bolt-magento2/docker-container/docker-format.py $phpVersion
+fi
+if [ "$installSampleData" = true ] ; then
+    php -dmemory_limit=5G bin/magento sampledata:deploy
```

Comment:
I advice to use instead of 
`php -dmemory_limit=5G bin/magento sampledata:deploy` 
the following 
`php bin/magento setup:performance:generate-fixtures setup/performance-toolkit/profiles/ce/small.xml`

It will generate categories, products and etc. 
It is more light and faster request than `sampledata:deploy`

Reply:
When attempting to run this I am getting the following error `Area code "adminhtml" does not exist`

Reply:
@ethanwayda22 in that step: In which state Magento is?
Magento already installed?

Reply:
@ethanwayda22 Can you insert it in docker-container/configure-store.sh ?
After you created an admin user.

Reply:
@YevhenShBolt I believe for now it is best to use the sample data because it is in line with the integration tests that we run. However, if you choose not to install sample data I can leave a note in the playbook to run this manually.